generic/nstress.py

```diff
@@ -54,24 +44,23 @@ def setUp(self):
         tarball = self.fetch_asset(url, expire='10d')
         archive.extract(tarball, self.workdir)
         self.duration = self.params.get('duration', default=300)
+        self.testcase = self.params.get('testcase', default='nmem')
         self.memchunk = self.params.get('memchunk', default="250")
-        self.memchunk_nmem64 = self.params.get('memchunk_nmem64',
-                                               default="2047")
         self.procs = self.params.get('procs', default="250")
 
     def test(self):
         os.chdir(self.workdir)
-        self.run_cmd("./nmem -m %s -s %s" % (self.memchunk, self.duration))
-        self.run_cmd("./nmem64 -m %s -s %s" %
-                     (self.memchunk_nmem64, self.duration))
-        if self.is_fail >= 1:
-            self.fail("nstress test failed")
+        if process.system('./%s -m %s -s %s' % (self.testcase,
+                                                self.memchunk, self.duration),
+                          ignore_status=True, sudo=True, shell=True):
+            self.fail("%s test failed " % self.test)
+
         ''' ncpu retrun code is 1 even after successful completion'''
-        ncpu_result = process.run("./ncpu -p %s -s %s" %
-                                  (self.procs, self.duration),
-                                  ignore_status=True, sudo=True)
-        if ncpu_result.exit_status != 1:
-            self.log.info("ncpu test failed")
+
+        if process.system("/ncpu -p %s -s %s" %
```

Comment:
@PraveenPenguin just noticed, it should be `./ncpu`

Reply:
ah thanks  i miss that while doing rebase : (